src/font-family-name-value.ts

```diff
@@ -0,0 +1,49 @@
+/**
+ * Copyright 2023 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { TokenType, tokenize } from '@csstools/css-tokenizer'
+
+export const fontFamilyNameValue = (string: string): string | undefined => {
+  const tokens = tokenize({ css: string }).filter(
+    x =>
+      x[0] !== TokenType.Comment &&
+      x[0] !== TokenType.Whitespace &&
+      x[0] !== TokenType.EOF,
+  )
```

Comment:
Maybe we can do something like this:

```ts
const tokens = tokenize({ css: string }).filter(([tokenType]) => !IGNORED_TOKEN_TYPES.has(tokenType))
```

With this outside the function:
```ts
const IGNORED_TOKEN_TYPES: ReadonlySet<TokenType> = new Set([TokenType.Comment, TokenType.Whitespace, TokenType.EOF])
```

Reply:
yes, that makes it clearer which bits are important, thanks